book/03-git-branching/sections/nutshell.asc

```diff
@@ -3,13 +3,13 @@
 
 To really understand the way Git does branching, we need to take a step back and examine how Git stores its data.
 
-As you may remember from <<ch01-getting-started#_what_is_git>>, Git doesn't store data as a series of changesets or differences, but instead as a series of _snapshots_.
+As you may remember from <<ch01-getting-started#section__what_is_git>>, Git doesn't store data as a series of changesets or differences, but instead as a series of _snapshots_.
```

Comment:
Did you mean to add 2 underscores here: `#section__`?

Reply:
The only other anchor tag in the book uses just 1 underscore after `#section_` so to be consistent I would only use 1 underscore. :smile:

Reply:
See: https://raw.githubusercontent.com/progit/progit2/master/book/04-git-server/sections/gitlab.asc

Reply:
Yes I meant to use 2 underscores to distinguish the 'type' from the 'identifier'.

But good catch with the other ID; I'll change it to be consistent (but then without a leading underscore); i.e. `what_is_git_section`

Reply:
@jnavila what do you think?

Reply:
I'm not purist, but just being consistent is great.